.github/workflows/tidy-modern.yml

```diff
@@ -0,0 +1,34 @@
+name: tidy-modern
+
+on: [push, pull_request]
+
+jobs:
+  tidy:
+    runs-on: ubuntu-latest
+    container:
+      image: pointcloudlibrary/env:18.04
```

Comment:
I would suggest that we update this to 20.04 or even 22.04

Reply:
22.04 it is